resources/views/admin/dashboards.blade.php

```diff
@@ -0,0 +1,91 @@
+@extends('layouts.app')
+
```

Comment:
create a section here like

```
@section('styles')
    // ... add the code from line number 7 and line number 8 here and remove them inside `content` section
@endsection
```

and...
in your app.blade.php

and `@yield('styles')` before the `</head>` end tag

Reply:
I did.